EXILED/Exiled.API/Features/Waves/TimedWave.cs

```diff
@@ -53,15 +52,27 @@ public class TimedWave
         public Faction Faction => timedWave.TargetFaction;
 
         /// <summary>
-        /// Gets the team of this wave.
+        /// Gets the spawnable faction for this wave.
         /// </summary>
-        public SpawnableTeamType Team => timedWave.TargetFaction.GetSpawnableTeam();
+        public SpawnableFaction SpawnableFaction => Faction switch
```

Comment:
You can also make this team too. GetSpawnableTeam() still exists, but it will return Team rather than SpawnableTeamType